### DISCUSSION TITLE
Using prerendering and pageContext.urlParsed

### DISCUSSION THREAD

**Discussion by philwinder** [vikejs/vike#914] [Discussion] [answered]
Hi there,
I'm trying to use `urlParsed` when prerendering. It works when in dev, but it was always null when prerendering so I started digging around. I found this line of code [that is commented out](https://github.com/brillout/vite-plugin-ssr/blob/main/vite-plugin-ssr/node/prerender/runPrerender.ts#L482). So I think that explains why the pageContext is different when prerendering.

Questions:
1. Why is this disabled? I.e. is there a good reason why it is disabled?
2. How can I reenable it?
3. I think the comment in the code is suggesting that I can use a `onPrerenderStart` function to reenable this. Should I re-add the `addComputedUrlProps` to that? Could you give an example?

Thanks, Phil

**Comment by brillout** [comment, marked as answer]
2784ac938e4840a7ffc679c077cf4ecd7cf5a3f7 should fix it. It's released in `0.4.129`.

Btw. in case you can convince your employer: https://github.com/sponsors/brillout.
